js/apps/admin-ui/maven-resources-community/theme/keycloak.v2/admin/messages/messages_de.properties

```diff
@@ -177,3 +202,438 @@ category=Kategorie
+addPolicy=Richtlinie hinzufügen
+otpPolicy=OTP Richtlinie
+webauthnPolicy=Webauthn Richtlinie
+webauthnPasswordlessPolicy=Webauthn Passwordless Richtlinie
+cibaPolicy=CIBA Richtlinie
+addExecutionTitle=Ausführungsschritt hinzufügen
+addExecution=Ausführung hinzufügen
+addSubFlowTitle=Unterablauf hinzufügen
+addSubFlow=Unterablauf hinzufügen
+emptyExecution=Keine Ausführungsschritte vorhanden
+emptyExecutionInstructions=Sie können mit der Definition dieses Ablaufs beginnen, indem Sie einen Teilablauf oder einen Ausführungsschritt hinzufügen.
+bindFlow=Ablauf verknüpfen
+editInfo=Ablauf editieren
+editFlow=Ablauf editieren
+addStep=Ausführungsschritt hinzufügen
+flowDetails=Ablaufdetails
+steps=Ablaufschritte
+requirement=Erforderlich
+authenticationAliasHelp=Name der Konfiguration
+authenticatorRefConfig.value.label=Authenticator-Referenz
+authenticatorRefConfig.value.help=Fügen Sie einen benutzerdefinierten Referenznamen für den Authenticator hinzu. Wenn dieser Authentifikator während eines Authentifizierungsablaufs erfolgreich abgeschlossen wird, verwendet der AMR-Protokoll-Mapper (Authentication Method Reference) diesen Wert, um den AMR-Anspruch der generierten Token aufzufüllen. Beachten Sie, dass das AMR-Protokoll für den jeweiligen Client konfiguriert sein muss, um den AMR-Anspruch aufzufüllen.
+authenticatorRefConfig.maxAge.label=Authenticator-Referenz  Maximales Alter
+authenticatorRefConfig.maxAge.help=Das maximale Alter in Sekunden, für das der Authenticator-Referenzwert in einer SSO-Sitzung gültig ist. Wenn der Protokoll-Mapper für die Authentifizierungsmethoden-Referenz (AMR) verwendet wird, wird die AMR nur dann als gültig angesehen und in das Token eingetragen, wenn die Ausführung des Authenticators innerhalb des angegebenen Höchstalters abgeschlossen wurde.
+identityProviders=Identitätsanbieter
+listExplain=Identitätsanbieter sind soziale Netzwerke oder Identitätsbroker, die es den Benutzern ermöglichen, sich bei Keycloak zu authentifizieren.
+getStarted=Wählen Sie einen Anbieter aus der unten stehenden Liste aus, um loszulegen.
+userFederation=Benutzerföderierung
+userFederationExplain=Die Benutzerföderation ermöglicht den Zugriff auf externe Datenbanken und Verzeichnisse wie LDAP und Active Directory.
+userEvents=Benutzereignisse
+emptyUserEvents=Keine Benutzerereignisse vorhanden
+emptyUserEventsInstructions=In diesem Realm gibt es keine Benutzerereignisse.
+eventExplain=Ereignisse sind Aufzeichnungen von Benutzer- und Adminereignissen in diesem Bereich. Um die Verfolgung dieser Ereignisse zu konfigurieren, gehen Sie zu <1>Ereignisse</1> in den Realm-Einstellungen.
+adminEvents=Adminereignisse
+exactSearch=Exakte Suche
+noGroupsInThisRealm=Keine Gruppen in diesem Realm vorhanden.
+noGroupsInThisRealmInstructions=Sie haben noch keine Gruppen in diesem Bereich erstellt. Erstellen Sie eine Gruppe, um loszulegen.
```

Comment:
Again "loszulegen". Maybe replace this globally in this file? I will ignore it from now on.

Reply:
Changed all occurences to "um zu beginnen."